File: java/Car.java
package POO.java;

public class Car {
    private Integer id;
    private String placa;
    private Account driver;
    private Integer passengers;


    public Car(String placa, Account driver){
        this.placa = placa;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("Placa: " + placa + " Conductor: " + driver.name);
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getPlaca() {
        return placa;
    }

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    public Integer getPassengers() {
        return passengers;
    }

    public void setPassengers(Integer passengers) {
        if(passengers == 4){
        this.passengers = passengers;
        }else{
            System.out.println("Son 4 cupos para este tipo de servicio");
        }
    }
    
}

File: php/car.php
<?php
require_once('account.php');
class Car{
    public $id;
    public $placa;
    public $driver;
    protected $passenger;

    public function __construct($placa,$driver)
    {
        $this->_placa=$placa;
        $this->_driver=$driver;
    }

    public function printDataCar(){
        echo "Licencia: $this->placa 
        Conductor: {$this->driver->name}
        Número de pasajeros: $this->passenger";
    }

    public function getPassenger() {
        return $this->passenger;
    }

    public function setPassenger() {
        if ($passenger ==4) {
            $this->passenger = $passenger;
        }
        else {
            echo "La capacidad es de 4 pasajeros";
        }
    }
}